runtime/lua/vim/filetype.lua

```diff
@@ -897,6 +897,12 @@ local extension = {
       return "text"
     end
   end,
+  cmd = function (path, bufnr)
+    if getline(bufnr, 1):match("^/\\*") then
```

Comment:
I don't think you can do that in a Lua pattern (which are not regular expressions)?

Reply:
(Did you test this?)

Reply:
Just lurking, but this is a valid pattern and matches the same text that that regex would match (a line starting  with `/` followed by zero or more `\`).

The original regex is `^/\*` though, which should be translated to `^/%*`.

Reply:
And should use `find` over `match`.